### PR TITLE
Support webpack multiple entry points and dynamic output filename

### PR DESCRIPTION
**Pull request by liunaijia**
This PR aims at adding support webpack multiple entry points and dynamic output filename which was discussed at https://github.com/ai/size-limit/issues/74.

### CODE REVIEW

**Comment by liunaijia** [comment]
When webpack config file is given in size-limit config neither `path` nor `entry` is needless. Because customized webpack `entry` will be used in such case.

So either `path` or `config` is required in size-limit config.
- When `path` is given, size-limit will use it as entry point, it's suitable for the applications without webpack config.
- When `config` is defined, size-limit will use the given webpack config file to bundle and measure the size of output files.

**Comment by ai** [comment]
> When config is defined, size-limit will use the given webpack config file to bundle and measure the size of output files.

What if user will want to check size of one entry from many?

File: test/index.test.js [hunk 1]
```diff
@@ -100,3 +100,60 @@ it('uses custom webpack config', () => {
...
+      expect.assertions(1)
+      return getSize(null, {
+        config: fixture(`webpack-multipe-entry-points/webpack.config`),
+        entry: 'moduleBad'
+      }).then(() =>
+        expect(true).toBe(false)
+      ).catch(error =>
+        expect(error.message).toBe(
+          'Cannot find entry points moduleBad, ' +
+          'available entry points are moduleA,moduleB,moduleC')
```
Comment (ai) [inline test/index.test.js]: `,` → `, ` in entries list

**Comment by ai** [comment]
Great. The last fixed:

You need to add a test in `cli.js` that `entry` could be loaded from Size Limit config.

And you can add validation for `entry` field in config https://github.com/ai/size-limit/blob/master/cli.js#L105

File: test/index.test.js (hunk 1 above)
Comment (liunaijia) [reply test/index.test.js]: Done.

**Comment by liunaijia** [comment]
@ai Thanks for your feedback, I've addressed them.

**Comment by ai** [comment]
Awesome. I will merge it today or tomorrow (sorry, need to prepare for conference)

**Comment by ai** [comment]
Released in 0.21. Thanks for work.

**Pull request merged**